From akpm@linux-foundation.org  Thu Jun 19 11:55:35 2008
From: Christophe Jaillet <christophe.jaillet@wanadoo.fr>
Date: Mon, 09 Jun 2008 16:39:53 -0700
Subject: USB: AccessRunner: avoid unnecessary memset
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org, akpm@linux-foundation.org, christophe.jaillet@wanadoo.fr
Message-ID: <200806092339.m59Ndrj7015010@imap1.linux-foundation.org>


From: Christophe Jaillet <christophe.jaillet@wanadoo.fr>

Remove an explicit memset(.., 0, ...) to a variable allocated with kzalloc
(i.e.  'card_info' array of the structure 'instance').

Signed-off-by: Christophe Jaillet <christophe.jaillet@wanadoo.fr>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/cxacru.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/atm/cxacru.c
+++ b/drivers/usb/atm/cxacru.c
@@ -1052,7 +1052,6 @@ static int cxacru_bind(struct usbatm_dat
 
 	instance->usbatm = usbatm_instance;
 	instance->modem_type = (struct cxacru_modem_type *) id->driver_info;
-	memset(instance->card_info, 0, sizeof(instance->card_info));
 
 	mutex_init(&instance->poll_state_serialize);
 	instance->poll_state = CXPOLL_STOPPED;
